src/test/rbf_tests.cpp

```diff
@@ -374,6 +374,9 @@ BOOST_FIXTURE_TEST_CASE(improves_feerate, TestChain100Setup)
     const auto tx2_fee = entry2->GetModifiedFee();
     const auto tx2_size = entry2->GetTxSize();
 
+    // conflicting transactions
+    const auto tx1_conflict = make_tx(/*inputs=*/ {m_coinbase_txns[0], m_coinbase_txns[2]}, /*output_values=*/ {10 * COIN});
```

Comment:
nit: these changes were perhaps meant for d7dc9fd2f7bc675256687b9c55fdbec9cc8ac781 instead of 284a1d33f1dcbc3b3404ea40a948ff6600239613?